src/database/queries/reviews.js

```diff
@@ -14,19 +16,27 @@ exports.findAll = async (filter, attributes = {}) => {
 		return error
 	}
 }
-exports.countDistinct = async (filter, attributes = {}) => {
+
+exports.countDistinct = async (filter, attributes) => {
```

Comment:
@adithyadinesh0412 it's a function to get the unique resource Id then use the custom name for this function